content/en/blog/2022/jaeger-native-otlp/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Supporting Jaeger with native OTLP
+linkTitle:
+date: 2022-10-20
+author: >-
+  [Jason Plumb](https://github.com/breedx-splk) (Splunk)
+draft: true
+---
+
+Back in May of 2022, the Jaeger project
+[announced native support for the OpenTelemetry Protocol](https://medium.com/jaegertracing/introducing-native-support-for-opentelemetry-in-jaeger-eb661be8183c)
+(OTLP). This followed a
+[generous deprecation cycle](https://twitter.com/YuriShkuro/status/1455170693197402119)
+for the Jaeger client libraries across many languages. With these changes,
+OpenTelemetry users are now able to send traces into Jaeger with
+industry-standard OTLP, and the Jaeger client library repositories have been
+finally archived.
+
+## OpenTelemetry Support
+
+This interoperability is a wonderful victory both for Jaeger users and for
+OpenTelemetry users. However, we're not done yet. The OpenTelemetry
+specification still requires support for Jaeger client exporters across
+languages.
+
+This causes challenges for both Jaeger users and OpenTelemetry maintainers.
+
+### #1. Confusing Choices
+
+Currently, users are faced with a choice of exporter (Jaeger or OTLP), and this
+can be a source of confusion. A user might be inclined, when exporting telemetry
+to Jaeger, to simply choose the Jaeger exporter because the name matches (even
+though Jaeger now actively encourages the use of OTLP).
+
+If we can eliminate this potentially confusing choice, we can improve the user
+experience and continue standardizing on a single interoperable protocol. We
+love it when things "just work" out of the box!
+
+### #2. Maintenance and Duplication
+
+Because the Jaeger client libraries are now archived, they will not receive
+updates (including security patches). To continue properly supporting Jaeger
+client exporters, OpenTelemetry authors would be required to re-implement some
+of the functionality it had previously leveraged from the Jaeger clients.
+
+Now that Jaeger supports OTLP, this feels like a step backwards: It results in
+an increased maintenance burden with very little benefit.
+
+## Intent to Deprecate
```

Comment:
Can we get a little of this, like a sentence, up near the top? Since the primary thing here is that there's an intent to deprecate, it should also be summarized at the top before an elaboration of existing jaeger support.